src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json

```diff
@@ -20,6 +20,7 @@
   "sensenet": {
     "authentication": {
       "authority": "https://localhost:44311",
+      "metadatahost": "http://SenseNet.IdentityServer4.Web",
```

Comment:
When do we need this? Only in a docker env, or always? Maybe the default should be an empty string.

Reply:
containers have to use different channels when communicate directly

Reply:
we can probably move this to launchsettings for Docker